lib/fastlane/plugin/wpmreleasetoolkit/actions/ios/ios_get_app_version.rb

```diff
@@ -22,7 +21,15 @@ def self.details
       end
 
       def self.available_options
-        # Define all options your action supports.
+        [
+          FastlaneCore::ConfigItem.new(
+            key: :public_version_xcconfig_file,
+            env_name: 'FL_IOS_GET_APP_VERSION_PUBLIC_VERSION_XCCONFIG_FILE',
```

Comment:
I anticipate that, when we will add a similar `ConfigItem` to more actions (in future PRs), we'll likely also name those `public_version_xcconfig_file`, while they will all expect the same `.xcconfig` file path each and won't really benefit from having a _separate/different_ ENV var for each separate action. Given that, what do you think about using an env var name that would be a bit more generic (and action-agnostic), like `FL_IOS_PUBLIC_VERSION_XCCONFIG_FILE`… that would _also_ fit the other actions when those will end up also adopting such `ConfigItem` later?

Reply:
Very good point! This env var will definitely be required on several actions and it refers to exactly the same thing -- I was trying to think of reasons to have different values between actions, but I don't think this is something that can (or even should) happen in practice.
I'll change the `env_name` to a more generic one and, when I touch another action later, I'll use the same name 👍